src/util.h

```diff
@@ -86,6 +86,7 @@ int dirRemove(char *dname);
 int fileExist(char *filename);
 sds makePath(char *path, char *filename);
 int fsyncFileDir(const char *filename);
+int invalidatePageCache(int fd, size_t offset, size_t length);
```

Comment:
We really don't want to `invalidate` page cache, we want to `reclaim` page cache here, So I think `reclaimFilePageCache` seems better here.